src/main/scala/ch/epfl/bluebrain/nexus/kg/resources/syntax.scala

```diff
@@ -45,4 +46,22 @@ object syntax {
                                                                       http: HttpConfig) {
     def accessId: AbsoluteIri = AccessId(res.id.value, res.schema.iri)
   }
+
+  final implicit class GraphSyntaxMeta(private val g: Graph) extends AnyVal {
+
+    /**
+      * Removes the metadata triples from the graph centered on the provided subject ''id''
+      *
+      * @param id the subject
+      * @return a new [[Graph]] without the metadata triples
+      */
+    def removeMetadata(id: IriOrBNode): Graph =
```

Comment:
I'm wondering if this should be instead defined in the `ResourceF` companion object as the metadata is really something controller by a Resource

Reply:
It is better used as part of the syntax. I can rename it to `removeResourceMetadata` if that makes it more clear, or make something explicit on the docs.

Reply:
I'm not arguing about the syntax, I think it's a good idea to provide syntax for the function, but the syntax can call the function in the companion object; this way we keep things that are strongly related together instead of spread around in various parts of the codebase.